### PR TITLE
Bar diagram for current probabilities

### LINKED ISSUE #203: Bar diagram for current probabilities
- In the market detail screen as well as the buy and sell outcomes screens it would be nice to have bar diagram indicating the (changing) probabilities. Could be behind the text like in sight: https://sight.pm/0x078905a51efb905e64465b944ecf993f6cc200fe10d681b5bebce38ccf7474f2/#/

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #203 

#### Includes
- Bar diagram component for current probabilities
- Applied to view, sell, buy and fund section
- Removed outcome column on some table display
- Here is some images ...
Screenshot_20191128_164720
Screenshot_20191128_164602
Screenshot_20191128_164638
Screenshot_20191128_164402

### CODE REVIEW

**Comment by gabitoesmiapodo** [comment]
It looks great.

I'd recommend that you create another issue where we could improve its looks a bit more with a new design.

But it's fine otherwise.

**Comment by mariano-aguero** [comment]
> It looks great.
> 
> I'd recommend that you create another issue where we could improve its looks a bit more with a new design.
> 
> But it's fine otherwise.

I create an issue for this https://github.com/protofire/gnosis-conditional-exchange/issues/249 , thanks for your feedback.

File: app/src/components/common/bar_diagram_probabilities/index.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React, { HTMLAttributes } from 'react'
+import styled from 'styled-components'
+
+interface StyleProps {
+  isGreaterThanFifty?: boolean
+  barPercentage?: number
```
Comment (fvictorio) [inline app/src/components/common/bar_diagram_probabilities/index.tsx]: `percentage`, the context is `bar` already.

File: app/src/components/common/bar_diagram_probabilities/index.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+export const BarDiagram: React.FC<Props> = (props: Props) => {
+  const { outcomeName, probability } = props
+
+  const isGreaterThanFifty = probability > 50
+
+  return (
+    <OutcomeBar>
+      <Probability isGreaterThanFifty={isGreaterThanFifty}>
```
Comment (fvictorio) [inline app/src/components/common/bar_diagram_probabilities/index.tsx]: Maybe we can make this a little more future proof and use something like `isWinning`? Of course, that should be something that receives from its parent.

File: app/src/components/common/bar_diagram_probabilities/index.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    props.isGreaterThanFifty ? props.theme.colors.primarySoft : props.theme.colors.secondarySoft};
+  color: #00193c;
+  display: block;
+  height: 32px;
+  padding-bottom: 4px;
+  padding-left: 2px;
+  padding-right: 2px;
+  padding-top: 4px;
+  text-align: center;
+  width: ${props => props.barPercentage || '50'}%;
```
Comment (fvictorio) [inline app/src/components/common/bar_diagram_probabilities/index.tsx]: Can the `percentage` be falsy?

Comment (mariano-aguero) [reply app/src/components/common/bar_diagram_probabilities/index.tsx]: Now percentage is not optional

File: app/src/components/common/bar_diagram_probabilities/index.tsx (hunk 2 above)
Comment (mariano-aguero) [reply app/src/components/common/bar_diagram_probabilities/index.tsx]: Sure, I update the variable name to `isWinning`, and is received from its parent

File: app/src/components/common/bar_diagram_probabilities/index.tsx (hunk 1 above)
Comment (mariano-aguero) [reply app/src/components/common/bar_diagram_probabilities/index.tsx]: Update `barPercentage` to `percentage`

**Pull request merged**